en/ios/user-interface.mdown

```diff
@@ -484,8 +484,56 @@ The easiest way to understand this class is with an example. This subclass of `P
 @end
 ```
 ```swift
-// no Swift example
-// Want to contribute to this doc? https://github.com/ParsePlatform/Docs
+class SimpleTableViewController : PFQueryTableViewController {
+    
+    override init(style: UITableViewStyle, className: String?) {
+        super.init(style: style, className: className)
+        self.parseClassName = "Todo"
+        self.pullToRefreshEnabled = true
+        self.paginationEnabled = true
+        self.objectsPerPage = 25
+    }
+
+    required init!(coder aDecoder: NSCoder!) {
```

Comment:
I think it's my old version of XCode which proposed me it this way. I'll change it because you're right (https://developer.apple.com/library/prerelease/ios/documentation/Cocoa/Reference/Foundation/Protocols/NSCoding_Protocol/index.html#//apple_ref/occ/intfm/NSCoding/initWithCoder:) the method signature is indeed `public init?(coder aDecoder: NSCoder)`